test/app/src/config.ts

```diff
@@ -4,11 +4,12 @@ const HOST = window.location.host;
 const PROTO = window.location.protocol;
 const REDIRECT_URI = `${PROTO}//${HOST}${CALLBACK_PATH}`;
 const POST_LOGOUT_REDIRECT_URI = `${PROTO}//${HOST}/`;
-const DEFAULT_SIW_VERSION = '5.2.3';
+const DEFAULT_SIW_VERSION = ''; // blank for local/npm/bundled version
```

Comment:
I feel like "bundled" is confusing (here and a few places) - it's not bundled so much as a hardcoded version pulled from CDN, right?

Reply:
No this is a version installed from yarn and bundled with webpack. If you specify a version it will pull from CDN, if you don't it will use internally bundled version. This allows `yarn link` for local dev.

Reply:
You've lost me - auth-js doesn't include SIW (and good, because SIW does include (bundle) auth-js).

Reply:
This is the test app

Reply:
the test app uses the signin widget to perform the complete flow. siw handles interaction_required events and auth-js will handle the login redirect / token exchange.  Basically the same as the samples, but uglier!